doc/Manuals/ug-cs_x-road_6_central_server_user_guide.md

```diff
@@ -1107,11 +1120,11 @@ An audit log record contains
 
 For example, adding a new member in the central server produces the following log record:
 
-`2015-07-03T11:40:52+03:00 my-central-server-host INFO  [X-Road Center UI] 2015-07-03 11:40:52+0300 - {"event":"Add member","user":"admin1", "data":{"memberName":"member1 name","memberClass":"COM", "memberCode":"member1"}}`
+`2023-05-21T16:20:06+03:00 my-central-server-host correlation-id: [655a2150c4688558] INFO  [X-Road Central Server Admin Service] 2023-05-21T16:20:06.267+03:00 - {"event":"Add member","user":"xrd","ipaddress":"192.0.2.1","auth":"Session","url":"/api/v1/members","data":{"memberName":"SS2 OWNER","memberClass":"TEST","memberCode":"SS2_OWNER"}}`
```

Comment:
The IP address of the user could be mentioned in the above list "An audit log record contains"